frontend/src/App.tsx

```diff
@@ -1,26 +1,24 @@
-import {LoginForm} from './components/loginForm';
-import {SignUp} from './components/signup';
-import { TempLandingPage } from './components/tempLandingPage';
-import './index.css';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import { Contacts } from './components/contacts';
-function App() {
+import { LoginForm } from './components/loginForm'
+import { SignUp } from './components/signup'
+import { TempLandingPage } from './components/tempLandingPage'
+import './index.css'
+import { Routes, Route } from 'react-router-dom'
+import { Contacts } from './components/contacts'
+import { PlanForm } from './components/PlanForm'
+import React from 'react'
+
+function App (): JSX.Element {
   return (
-<div> 
-    <Routes> 
-      <Route path = "/login" element = {<LoginForm/>}> </Route>
-    </Routes>
-    <Routes> 
-      <Route path = "/signup" element={<SignUp/>}>  </Route>
-    </Routes>
-    <Routes>
-      <Route path = "/tempLandingPage" element={<TempLandingPage/>}> </Route>
-    </Routes>
-    <Routes>
-      <Route path = "/contactPage" element={<Contacts/>}> </Route>
-    </Routes>
-</div>
-  );
+    <div>
+      <Routes>
+        <Route path="/login" element={<LoginForm />} />
+        <Route path="/signup" element={<SignUp />} />
+        <Route path="/tempLandingPage" element={<TempLandingPage />} />
+        <Route path="/contactPage" element={<Contacts />} />
+        <Route path="/planForm" element={<PlanForm />} />
+      </Routes>
+    </div>
+  )
 }
 
-export default App;
```

Comment:
maybe it's just me being old school but I'm team require semicolon. if you are too, might be worth updating the eslint config and adding a rule override that restores the semicolon rule